src/common/pageable.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2020 BlockDev AG
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { validateMultiple } from '../fmt/validation'
+
+export interface Pageable {
+  items: any
+  page: number
+  pageSize: number
+  totalItems: number
+  totalPages: number
+}
```

Comment:
```suggestion
export interface Pageable<T> {
  items: T
  page: number
  pageSize: number
  totalItems: number
  totalPages: number
}
```

Reply:
This would make Pageable generic, and anything that extends it should be forced to define `T`